content/lxd/getting-started-cli.md

```diff
@@ -78,17 +78,17 @@ To install the feature branch of LXD, run:
 
 #### Ubuntu
 ##### Ubuntu (all releases)
-The recommended way to install LXD is with the snap.
+The recommended way to install LXD is the [snap package](https://snapcraft.io/lxd).
 
-For the latest stable feature release, use:
+For the feature release, use:
 
     sudo snap install lxd
 
-For the LXD 4.0 stable release, use:
+For the LXD 4.0 LTS release, use:
```

Comment:
We should define what "LTS" means in this document. Perhaps in the section I suggest we add above that explains the difference between the feature release and the LTS release channels.

Reply:
I'd also add the word "latest", so its "latest LXD 4.0 LTS release" to align with the suggested terminology above and the snap channel actually being used.

Reply:
1. There is already an explanation here: https://linuxcontainers.org/lxd/getting-started-cli/#choose-your-release

2. I deleted the word `latest`, because people might get confused by it.
I think that people will connect `latest` with the newest available release, which would be the feature release.
Also it should be kind of self-evident that the latest version of a branch will be installed.
But if you insist on it, it's of course your decision.

Reply:
I think I agree that `latest` here makes a bit less sense since it doesn't line up with the channel name. so just `4.0 LTS release` should be fine especially once we have the expanded LTS term up at the top.

Reply:
Ok, then I will keep it this way.